data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	Tavish watches him go. "I dare say he didn't deserve additional recompense after what he did to us."`
+			`	Kormak grunts in agreement.`
+				accept
+
+
+	on visit
+		dialog `You haven't found the Broken Record yet.`
+	on complete
+		conversation
+			`Your ship lands in the heart of the continent-consuming megatropolis Polyphron, the "only city" on Hephaestus.`
+			`	Finding your way around here could well be impossible. Thankfully Tavish says he knows the area where recently impounded ships are housed.`
+			`	He and Kormak head toward the spaceport.`
+
+
+mission "Tavish and Kormak 4"
+	name "A journey to Chiron"
+	description "Transport Tavish and Kormak to <planet> to look for a man named Doctor-Professor Dylan Hullen-Freeze."
+	source "Hephaestus"
+	destination "Chiron"
+	passengers 2
+	to offer
+		has "Tavish and Kormak 3: done"
+	on offer
+		conversation
+			`	You get the feeling the sun never sets on Hephaestus as you navigate droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of the human expansion, and not Earth.`
+			`	Tavish leads you down one narrow corridor filled with endless ships. You don't see any labels or signage that make sense, but somehow Tavish plows on with a quick step and unshakable confidence.`
+			`	Eventually he leads you into a large administrative complex. After four flights of stairs, three different elevators, and one very small trolley the three of you arrive at a bland corporate office with the words, "Repository of Confiscated Goods."`
+			`	There are no doors, and a single window that looks like it was made of reinforced glass. The tired looking agent on the other side of the glass hits a button and his voices buzzes over to you, "What can I help you with?"`
+			choice
+				`	"We're looking for stolen scientific research equipment that was recently confiscated aboard the Broken Record."`
+			`	The man looks down at his monitor and begins typing. After a while he stops and looks back up at you.`
+			`	"Ah yes, I found it. You'll be happy to know we did indeed confiscate some unusual equipment from the Broken Record." He looks at you expectantly.`
+			choice
+				`	"Great! Can it be returned to us?"`
+				`	"Finally. We're here to collect it."`
+				`	"Hand it over. It belongs to us."`
+			`	The man gives you a pained expression, "I am sorry, but it is no longer on Hephaestus."`
+			`	He says nothing else and waits.`
+			choice
+				`	"Okay. Where is it?"`
+				`	"Tell me where it is, right now."`
+				`	"Speak up! Where's our stuff?!"`
+			`	The agent's smile does not reach his eyes, "I am sorry, but that information is confidential."`
+			choice
+				`	"How are we supposed to reclaim our stolen goods?"`
+				`	"What do we have to do to get that information?"`
+				`	"Help me out here, what am I supposed to do?"`
```

Comment:
They player doesn't know that they don't do anything. Also, this is a conversation you're having - not them.

Reply:
The problem is that I believe this will just display one line of text at the top of the screen, then the menu, and hide the whole conversation you're having unless you scroll up.